src/main/java/frc/robot/subsystems/gripper/GripperIO.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.units.*;
+
+public interface GripperIO {
+
+    GripperInputsAutoLogged inputs = new GripperInputsAutoLogged();
+
+    void setRollerMotorPower(double power);
+
+    void setAngleMotorPower(double power);
+
+    void setAngle(MutableMeasure<Angle> angle);
+
+    void updateInputs(GripperInputs inputs);
```

Comment:
No need to have the inputs as a parameter, it exists as a member in the implementations.

Reply:
a554e0b5cf343ecb191494bf7cbf3e4f4c1c3394